### DISCUSSION TITLE
Please add documentation to show how to use docker to serve the project created with famework-x

### DISCUSSION THREAD

**Discussion by vikbert** [clue/framework-x#162] [Q&A] [answered]
First of all, thx for the awesome work.

Could you please add documentation to show how to use docker to serve the project created with famework-x ?

**Comment by clue** [comment, marked as answer]
@vikbert Glad you like X!

Have you seen the documentation for using Framework X with Docker containers already? https://framework-x.org/docs/best-practices/deployment/#docker-containers

If so, is there anything you're missing in particular? If not, please let me know how we can make sure it's easier to find for the next person stumbling upon this question. I hope this helps! :+1:

**Reply by vikbert** [reply]
Awesome I haven't seen that before. It seems so easy thanks for the link. P.s. the performance by using framework-x is impressive 👍🏻

**Reply by clue** [reply]
@vikbert Thank you very much for your kind words, makes me think we should almost set this up as a testimonial on the front page! 😃
